docs/solution-pack-guide.md

```diff
@@ -0,0 +1,27 @@
+# Usage of SOC Simulator Solution Pack
+
+## Configuring the FortiSOAR SOC Simulator Connector
+
+This solution pack will install and pre-configure the FortiSOAR SOC Simulator Connector by default. 
```

Comment:
Please mention the configuration name as well here

Reply:
Done